src/core/core.datasetController.js

```diff
@@ -268,6 +268,49 @@ helpers.extend(DatasetController.prototype, {
 		}
 	},
 
+	/*
+	 * Returns a set of predefined style properties that should be used to represent the dataset
+	 * or the data if the index is specified
+	 * @params {number} index - data index
+	 * @return {IStyleInterface} style object
+	 */
+	getStyle: function(index) {
+		var me = this;
+		var meta = me.getMeta();
+		var element, style;
+
+		if (meta.dataset && index === undefined) {
+			element = meta.dataset;
+		} else {
+			index = index || 0;
+			element = meta.data[index];
+		}
+		style = me._resolveElementOptions(element || {}, index);
+
+		if (style.fill === false || style.fill === null) {
+			style.backgroundColor = 'rgba(0,0,0,0)';
+		}
+
+		return {
```

Comment:
I think its safe to just `return style` here?

Reply:
I explicitly list the properties because we are using `IStyleInterface` as a return value.

Reply:
Where is this `IStyleInterface` defined and why should we care in javascript?

Reply:
It’s not defined in Chart.js code and I thought some TypeScript code may use it. I was not quite sure if we need to care this in JS code.

Reply:
Made the suggested change